src/renderers/testing/stack/ReactTestMount.js

```diff
@@ -138,6 +139,20 @@ ReactTestInstance.prototype.unmount = function(nextElement) {
   });
   this._component = null;
 };
+ReactTestInstance.prototype.toTree = function() {
+  const component = this._component._renderedComponent;
+  const element = component._currentElement;
+  // not using `children`, but I don't want to rewrite without destructuring
+  // eslint-disable-next-line no-unused-vars
+  const { children, ...propsWithoutChildren } = element.props;
+  return {
+    nodeType: ReactNodeTypes.getType(element),
```

Comment:
I thought about this. I feel like there are two ways of looking at this, and I'm not sure which one is more correct:

1. we just export a `nodeType` that makes the most sense based on the data structure available and not relying on the knowledge of the RST node spec.
2. we export a `nodeType` that is identified by the RST node spec.

The former is easier in terms of the react implementation and makes the "enzyme adapter" more complicated. The latter makes the enzyme adapter simpler, but means that react has to have some knowledge of this "RST node spec".

I'm open to either. Now that I'm saying it out loud, option 2 seems a bit better to me.